thinc/tests/layers/test_layers_api.py

```diff
@@ -128,7 +128,8 @@ def assert_data_match(Y, out_data):
     # ("CauchySimilarity.v1", {}, (array2d, array2d), array1d),
     ("ParametricAttention.v1", {}, ragged, ragged),
     ("SparseLinear.v1", {}, (numpy.asarray([1, 2, 3], dtype="uint64"), array1d, numpy.asarray([1, 1], dtype="i")), array2d),
-    ("remap_ids.v1", {"dtype": "f"}, ["a", 1, 5.0], array2dint)
+    ("remap_ids.v1", {"mapping_table": {}, "dtype": "int64"}, ["a", 1, 5.0], array2dint),
```

Comment:
I'd be a fan of keeping the existing test as is, then add new/additional ones as well.